### ISSUE TITLE
Add missing attributes to link component

### ISSUE THREAD

**Issue by bddjong** [nl-design-system/denhaag#598]
The link component is missing an aria-label attribute option. In the portal, this is causing an issue where a link with only an icon cannot be meaningfully understood when using a screenreader. 
![image](https://user-images.githubusercontent.com/24610692/145216591-a1c1d0d4-803e-41c1-b6e7-22f9582269fd.png)
